models/migrations/v180.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"xorm.io/xorm"
+)
+
+func addWhitespaceBehaviorUserColumn(x *xorm.Engine) error {
+	type User struct {
+		WhitespaceBehavior bool `xorm:"NOT NULL DEFAULT false"`
```

Comment:
Why is this different than in actual user struct?

Reply:
@lafriks its like https://github.com/go-gitea/gitea/blob/master/models/migrations/v141.go
And i think, sync full actual struct could possibly corrupt existing data (im not sure, but).